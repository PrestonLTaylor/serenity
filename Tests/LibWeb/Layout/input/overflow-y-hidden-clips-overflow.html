<style>
.y {
  height: 0;
  overflow-y: hidden;
}
</style><div class="y"><p>hidden</p></div>
