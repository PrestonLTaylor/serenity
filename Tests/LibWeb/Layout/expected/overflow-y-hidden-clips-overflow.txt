Viewport <#document> at (0,0) content-size 800x600 children: not-inline
  BlockContainer <html> at (0,0) content-size 800x600 [BFC] children: not-inline
    BlockContainer <body> at (8,8) content-size 784x0 children: not-inline
      BlockContainer <div.y> at (8,8) content-size 784x0 [BFC] children: not-inline
        BlockContainer <p> at (8,24) content-size 784x17.46875 children: inline
          line 0 width: 47.90625, height: 17.46875, bottom: 17.46875, baseline: 13.53125
            frag 0 from TextNode start: 0, length: 6, rect: [8,24 47.90625x17.46875]
              "hidden"
          TextNode <#text>
      BlockContainer <(anonymous)> at (8,8) content-size 784x0 children: inline
        TextNode <#text>
